standalone/docusaurus/src/components/HomepageFeatures.js

```diff
@@ -1,11 +1,14 @@
+import Bicycles from '@site/static/img/bicycles.png';
+import Laptop from '@site/static/img/laptop.png';
+import Utrecht from '@site/static/img/utrecht.png';
```

Comment:
By importing images like this, it doesn't break when the BaseUrl is changed at any point